packages/editor-ui/src/components/About.vue

```diff
@@ -1,35 +1,37 @@
 <template>
 	<span>
-		<el-dialog class="n8n-about" :visible="dialogVisible" append-to-body width="50%" title="About n8n" :before-close="closeDialog">
+		<el-dialog class="n8n-about" :visible="dialogVisible" append-to-body width="50%" :title="$translateBase('about.aboutN8n')" :before-close="closeDialog">
 			<div>
 				<el-row>
 					<el-col :span="8" class="info-name">
-						n8n Version:
+						{{ $translateBase('about.n8nVersion') }}:
 					</el-col>
 					<el-col :span="16">
 						{{versionCli}}
 					</el-col>
 				</el-row>
 				<el-row>
 					<el-col :span="8" class="info-name">
-						Source Code:
+						{{ $translateBase('about.sourceCode') }}:
 					</el-col>
 					<el-col :span="16">
 						<a href="https://github.com/n8n-io/n8n" target="_blank">https://github.com/n8n-io/n8n</a>
 					</el-col>
 				</el-row>
 				<el-row>
 					<el-col :span="8" class="info-name">
-						License:
+						{{ $translateBase('about.license') }}:
 					</el-col>
 					<el-col :span="16">
-						<a href="https://github.com/n8n-io/n8n/blob/master/packages/cli/LICENSE.md" target="_blank">Apache 2.0 with Commons Clause</a>
+						<a href="https://github.com/n8n-io/n8n/blob/master/packages/cli/LICENSE.md" target="_blank">
+							{{ $translateBase('about.apacheWithCommons20Clause') }}
+						</a>
 					</el-col>
 				</el-row>
 
 				<div class="action-buttons">
 					<el-button type="success" @click="closeDialog">
-						Close
+						{{ $translateBase('about.close') }}
```

Comment:
I extracted all the messages and ranked them by repetition - see [here](https://gist.github.com/ivov/8b8edadaf19ac5bfb4ce62ae46236af0).

Only 5 messages are repeated > 3 times.

- The most repeated message `''` should be excluded because it happens to be empty, i.e. not being used yet.
- `Default - ({defaultValue})` is used in a single component, so it does not make sense to extract it out.

So only 3 messages remain: `Save` (7), `Cancel` (4) and `Name` (4). `Cancel` should actually be 5 because of the overlooked key you found below.

Is it worth adding a special category for only 3 messages?

Reply:
Yes, `Save`, `Cancel` and `Name` to start with.